src/lidar/lidar_controller/lidar_controller.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
+from src.lidar.device_adapter.rplidar_adapter import RplidarAdapter
+from src.lidar.handler.display_handler import DisplayHandler
+from src.simulation.controller.probe import SimulationProbe
+from time import sleep
+from src.logger import LOGGER
+import threading
+
+
+class LidarController:
+
+    def __init__(self, rplidar_adapter: RplidarAdapter, display_handler: DisplayHandler,
+                 simulation_probe: SimulationProbe):
+        self.__rpa = rplidar_adapter
+        self.__dh = display_handler
+        simulation_probe.attach("position_obstacles", lambda: self.__rpa.get_seen_catesian())
+        self.start_thread_lidar_controller()
+
+    def __init_controller(self):
+        self.__rpa.init_lidar()
+        self.__dh.init_plot()
+
+    def __loop(self, display: bool):
+        try:
+            for scans in self.__rpa.generator_lidar_scans():
+                sleep(0.01)
```

Comment:
Il est là pour quoi le sleep?

Reply:
Pour rien, juste histoire d'avoir quelque chose dans le for